src/objects-table/ObjectsTable.js

```diff
@@ -199,12 +199,17 @@ class ObjectsTable extends React.Component {
     };
 
     getPaginationProps() {
-        const currentlyShown = calculatePageValue(this.state.pager, this.props.pageSize);
         const { pager } = this.state;
 
+        const isValidPage = _(["page", "total"])
+            .difference(_.keys(pager || {}))
+            .isEmpty();
+        if (!isValidPage) return {};
+        const pageCount = Math.ceil(pager.total / pager.pageSize);
```

Comment:
It should be this.props.pageSize, right?

Reply:
Right! It was working for me because I do have pageSize also on the pager, but should get from props. I'll push to dev.